src/main/java/com/autotune/experimentManager/core/ExperimentTrialHandler.java

```diff
@@ -44,6 +47,19 @@ public class ExperimentTrialHandler {
 
     public ExperimentTrialHandler(ExperimentTrial experimentTrial) {
         this.experimentTrial = experimentTrial;
+        String experimentName = experimentTrial.getExperimentName();
+        String trialNum = String.valueOf(experimentTrial.getTrialInfo().getTrialNum());
+        ConcurrentHashMap<String, HashMap<String, ExperimentTrial>> expTrailMap = EMMapper.getInstance().getExpTrialMap();
+        if (expTrailMap.containsKey(experimentName)) {
+            HashMap<String, ExperimentTrial> expMap = expTrailMap.get(experimentName);
+            if (!expMap.containsKey(trialNum)) {
+                expMap.put(trialNum, experimentTrial);
+            }
+        } else {
+            HashMap<String, ExperimentTrial> trialMap = new HashMap<String, ExperimentTrial>();
+            trialMap.put(trialNum, experimentTrial);
+            expTrailMap.put(experimentName, trialMap);
+        }
     }
 
     public static JSONObject getDummyMetricJson(ExperimentTrial experimentTrial) {
```

Comment:
Can you populate dummy values by calling empty abstract functions for each warm up and measurement cycles .
so tomorrow We just have to modify empty abstract functions to get real metric values. 
Look for comment here.
`        this.experimentTrial.getTrialDetails().forEach((tracker, trialDetails) -> {
            trialDetails.getPodContainers().forEach((imageName, podContainer) -> {
                podContainer.getTrialConfigs().forEach((trialNumber, containerConfigData) -> {
                    DeploymentHandler deploymentHandler = new DeploymentHandler(
                            trialDetails.getDeploymentNameSpace(),
                            trialDetails.getDeploymentName(),
                            containerConfigData);
                    IntStream.rangeClosed(1, numberOFIterations).forEach(
                            i -> {
                                deploymentHandler.initiateDeploy();
                                //check if load applied to deployment
                                //collect warmup and measurement cycles metrics
                            }
                    );
                });
            });
        });`